cypress/integration/manifest.js

```diff
@@ -11,6 +11,6 @@ describe('Manifest.json files', () => {
   ['news'].forEach(testManifest200s);
 });
 
-describeForLocalOnly('Local Env - Manifest.json files', () => {
+describeForLocalAndTest('Local Env - Manifest.json files', () => {
```

Comment:
These aren't on the Test env yet. 
```suggestion
describeForLocalOnly('Local Env - Manifest.json files', () => {
```